src/opnsense/www/index.php

```diff
@@ -58,27 +58,23 @@ function view_html_safe($text)
 }
 
 try {
-    /**
-     * Read the configuration
-     */
     $config = include __DIR__ . "/../mvc/app/config/config.php";
-
-    /**
-     * Read auto-loader
-     */
     include __DIR__ . "/../mvc/app/config/loader.php";
 
-    /**
-     * Read services
-     */
-    include __DIR__ . "/../mvc/app/config/services.php";
-
-    /**
-     * Handle the request
-     */
-    $application = new \Phalcon\Mvc\Application($di);
+    $router = new OPNsense\Mvc\Router('/ui/');
+    try {
+        $response = $router->routeRequest($_SERVER['REQUEST_URI'], [
+            'controller' => 'indexController',
+            'action' => 'indexAction'
+        ]);
+    } catch (\OPNsense\Mvc\Exceptions\DispatchException) {
+        // unroutable (page not found), present page not found controller
+        $response = $router->routeRequest('/ui/core/index/index');
```

Comment:
`/ui/core/index/index` could be misplaced here... looks too specific and hardcoded for this level